pylint_pytest/checkers/fixture.py

```diff
@@ -25,6 +25,19 @@ class FixtureChecker(BasePytestChecker):
             'deprecated-pytest-yield-fixture',
             'Used when using a deprecated pytest decorator that has been deprecated in pytest-3.0'
         ),
+        'W6402': (
+            'Using useless `@pytest.mark.usefixtures` decorator for fixtures',
+            'useless-pytest-usefixtures-decorator',
+            (
+                '@pytest.mark.usefixtures decorator doesn\'t work as expected for fixtures. '
+                'Take a look at: https://docs.pytest.org/en/stable/fixture.html#usefixtures'
```

Comment:
According to https://docs.pytest.org/en/stable/reference.html#marks, all `@pytest.mark` are not supported on fixtures.